corehq/apps/analytics/tasks.py

```diff
@@ -393,3 +396,9 @@ def _log_response(data, response):
         logger.error(message)
     else:
         logger.debug(message)
+
+
+def _get_ab_test_properties(user):
```

Comment:
or actually you can also just include the dict key as part of the input to `deterministic_random`. also is email guaranteed to be set? username or user_id might be better.